cluster/grpc_rpc_client.go

```diff
@@ -106,7 +108,7 @@ func (gs *GRPCClient) Call(
 
 	parent, err := tracing.ExtractSpan(ctx)
 	if err != nil {
-		logger.Log.Warnf("failed to retrieve parent span: %s", err.Error())
+		logger.Log.Warnf("[grpc client] failed to retrieve parent span: %s", err.Error())
```

Comment:
Shouldn't "grpcClient" be a field in the structured logging instead of a text?

Reply:
Yeah, ideally it should be, but pitaya's Logger interface doesn't expose the option to add fields :/

Reply:
Oh noes,`[grpc client]` looks great then. Maybe something to improve in the future?

Reply:
Agreed